frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -674,6 +684,11 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
             prop: 'catalogSources',
             optional: true,
           },
+          {
+            kind: referenceForModel(PackageManifestModel),
```

Comment:
Package manifests are expensive to watch. It's a fair amount of data and could noticeably slow down page load.

Is this simply to hide Subscriptions when there isn't a CSV yet? We should discuss if it's worth the extra complexity. It's a transient state (unless there is a problem creating the CSV, in which case you probably want to see the Subscription).

We might step back and look at this generally (as a separate PR). The experience is a little weird when installing an operator since the Subscription appears briefly and disappears. In some cases, there are briefly two rows for the same operator. @tlwu2013 can probably give more background on this. I assume showing the Subscription is for troubleshooting when the CSV can't be created.

Reply:
> Is this simply to hide Subscriptions when there isn't a CSV yet? 

The intention here is not for "hiding Subscriptions when there isn't a CSV yet" (but we do have another bug tracking that issue, but that one probably doesn't need to check `PackageManifest`object. And agree that PR could be a generic fix like you suggested).

The idea is "if the Operator has been marked as 'non-standalone' in its CSV file, then UI **doesn't show** Subscription object **unless** in error". Basically allowing `Subscription` object to mimic the "hidden" effect for the `CSV`object as well, so users won't see the those 'non-standalone' Operator showed up for certain period of time during the installation process (because of the `Subscription` object) and then be hidden afterward.

@spadgett in this case, is there another suggestion instead of like we thought here using the status stanza of `PackageManifest` object to get the annotation in the CSV file?

Reply:
@spadgett @tlwu2013 so in term of the PR we want (in the Installed Operators page) to hide all the Subscriptions and only show those that are in one of [these](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/vendor/github.com/operator-framework/api/pkg/operators/subscription_types.go#L17-L20) states. Right? That way we can remove listing on Package Manifests. Am I missing something ?